WcaOnRails/app/webpacker/javascript/wca-live/attempts.js

```diff
@@ -6,6 +6,9 @@ export const DNS_VALUE = -2;
  * Converts centiseconds to a human-friendly string.
  */
 function centisecondsToClockFormat(centiseconds) {
+  if (centiseconds == null) {
+    return '?:??:??';
```

Comment:
Yes -- the competition linked in the issue I filed (https://www.worldcubeassociation.org/competitions/GermanOpen2007/results/all?event=333mbo) has many results without a time.  I'll send this to WRT (though I'll wait till this PR goes live since it'll be clearer then :) )